### DISCUSSION TITLE
Error: Command "yarn run build" exited with 1

### DISCUSSION THREAD

**Discussion by abdurahman-dev** [vercel/community#160] [Help] [answered]
I am trying to  DEPLOY  my react app and I got some error
![image](https://user-images.githubusercontent.com/71510113/145343580-5cc94f3c-6a53-4104-a25e-5985aeb0c400.png)
![image](https://user-images.githubusercontent.com/71510113/145343645-4e53e072-82b9-458b-a4f1-8cc0a352eb15.png)

**Comment by Pieparker** [comment, marked as answer]
I'm sure you've found the answer already but [voila](https://github.com/vercel/community/discussions/30)! Warnings are being treated as errors and, because there are errors, the build has failed.

If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.

Remember: `Error: Command "yarn run build" exited with 1` is a **symptom** of a failed build and **not** the cause!
